src/app/[countryCode]/(main)/candles/page.tsx

```
@@ -1472,7 +1473,36 @@ const Candles = () => {
-            {/* Progress Scroll Bar Removed */}
+            {/* Progress Scroll Bar - Slider Style */}
+            <div
+              className="flex justify-center items-center w-full pt-3"
+              style={{ paddingTop: "1.5rem", paddingBottom: "20px" }}
+            >
+              <div
+                ref={sliderRef}
+                className="relative w-1/2 md:w-2/5 lg:w-1/3 h-0.5 bg-black/10 rounded-full cursor-pointer select-none group"
+                onClick={handleSliderClick}
+              >
+                {/* Slider Thumb - Moves left/right based on scroll */}
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)